lib/rucio/client/didclient.py

```diff
@@ -364,6 +364,24 @@ def list_files(self, scope, name, long=None):
             exc_cls, exc_msg = self._get_exception(headers=r.headers, status_code=r.status_code, data=r.content)
             raise exc_cls(exc_msg)
 
+    def bulk_list_files(self, dids):
```

Comment:
```suggestion
    def bulk_list_files(self, dids: list[dict[str, Any]]):
```

I'm not 100% sure what the return type would be here - I think it would be `Iterator[dict[str, Any]]`, but I'm not sure what the return type of `json.loads` would be in this context.